server/api/users/posts/[id].getposts.get.ts

```diff
@@ -0,0 +1,39 @@
+import { Posts } from "../../../dbModels";

```

Comment:
the name of file looks incorrect since with the suffix we are alreday aware that the api method is get we dont have explicitly add get in prefix "getposts"

Reply:
changed it to getPosts.ts